static/src/js/components/tournament/basic_info/basic_info_index.js

```diff
@@ -15,7 +15,9 @@ const mobx = require('mobx');
 export default class BasicInfo extends React.Component {
   // eslint-disable-next-line
   render() {
-    const tournament = mobx.toJS(this.props.stores.TournamentStore.tournament);
+    const tournament1 = mobx.toJS(this.props.stores.TournamentStore.tournament);
+
```

Comment:
const tournament = mobx.toJS(this.props.stores.TournamentStore.tournament) ? mobx.toJS(this.props.stores.TournamentStore.tournament) : ' ' ;